R/new-blog-post.R

```diff
@@ -0,0 +1,82 @@
+#' Create a new blog post
+#'
+#' Creates (and potentially opens) the `index.qmd` file for a new blog post.
+#'
+#' @param title A character string for the title of the post. It is converted
+#' to title case via [tools::toTitleCase()].
+#' @param dest A character string (or NULL) for the path within `posts`. By
+#' default, the title is adapted as the directory name.
+#' @param open A logical: have the default editor open a window to edit the
+#' `index.qmd` file?
+#' @param call A call object for reporting errors.
```

Comment:
My understanding is that the prefered way is to use
````
⁠@inheritParams rlang::args_error_context
````
From https://rlang.r-lib.org/reference/args_error_context.html and usage at several tidyverse packages